### ISSUE TITLE
EDSC-875: Collapsed collection details flyouts overlap for collections with three lines of title

### ISSUE THREAD

**Issue by earthdata-tasktop** [nasa/edsc-search-test-config-3#69]
See attached screen shot.  
The flyouts line up with the collapsed collection view, except if the collection name is 3 lines long, in which case the flyouts overlap and some flyout information is obscured.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

see attached screen shot for additional overlap examples (on Safari 8.0.3)

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

(Won't fix). There is no reasonable fix to this situation, but there is an easy workaround for users (close the overlapping popup). Further, it's not so much a bug as a UI constraint. HTML cannot automatically grow it horizontally, and making it wider further defeats some of the purpose of having the condensed view. Clipping it vertically causes the very problem we were trying to avoid with this design (cutting off collection ids).

I'll note this as feedback for design, but I don't see a clear place for this to go, unless you have ideas.

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

For this ticket, I marked the situation as "Won't Fix" based on Bo's description and having had a look at the first screenshot. I brought it up at sprint planning, wherein someone pointed out the second screenshot, and we kicked it back to development. The status now is that we have indeed fixed it. The fix was to remove the short name from the flyout. The situation in the first screenshot may still be present, and we had agreed we wouldn't address it, but the situation in the second screenshot no longer occurs.

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

(Won't fix). There is no reasonable fix to this situation, but there is an easy workaround for users (close the overlapping popup). Further, it's not so much a bug as a UI constraint. HTML cannot automatically grow it horizontally, and making it wider further defeats some of the purpose of having the condensed view. Clipping it vertically causes the very problem we were trying to avoid with this design (cutting off collection ids).

I'll note this as feedback for design, but I don't see a clear place for this to go, unless you have ideas.

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

The problem in the second screen shot (overlapping words) was addressed by removing the shortname from the flyout header. 

I looked at the same collections in UAT as in the second screenshot, and verified that the shortname was removed and that there were no overlapping words.  The problem in the first screen shot (overlapping flyout boxes) still occurs.

See attachment.

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

The problem in the second screen shot (overlapping words) was addressed by removing the shortname from the flyout header. 

I looked at the same collections in UAT as in the second screenshot, and verified that the shortname was removed and that there were no overlapping words.  The problem in the first screen shot (overlapping flyout boxes) still occurs.

See attachment.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

The problem in the second screen shot (overlapping words) was addressed by removing the shortname from the flyout header. 

I looked at the same collections in UAT as in the second screenshot, and verified that the shortname was removed and that there were no overlapping words.  The problem in the first screen shot (overlapping flyout boxes) still occurs.

See attachment.
